src/JSONSchema/logic/refHandlers.ts

```diff
@@ -0,0 +1,150 @@
+import { JSONSchemaType, IDSchemaPair } from '../types'
+import { getSplitPointer, concatFormPath } from './pathUtils'
+
+const isAbsoluteURI = (uri: string) => {
+  const absoluteRegExp = new RegExp(/^[a-z][a-z0-9+.-]*:/, 'i')
```

Comment:
You can make this `RegExp` a top-level variable to prevent it being creating over and over again

Reply:
Also: 
```suggestion
  const absoluteRegExp = /^[a-z][a-z0-9+.-]*:/i
```